src/frontend/next/public/menu.svg

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
Can we not use an icon from Material UI's Icon set instead of pulling in our own?  See https://material-ui.com/components/material-icons/ and search for `menu`.

Reply:
Imported Material UI's menu icon and placed it in the header.js component. So I believe we can actually delete Logo.svg now

Reply:
@humphd ^

Reply:
done I believe